commands/failures

```diff
@@ -0,0 +1,3 @@
+TestMinerCreate
```

Comment:
Are these notes? Probably shouldn't be here.

Reply:
Yes sorry, these will be removed.